### DISCUSSION TITLE
publish breadcrumbs

### DISCUSSION THREAD

**Discussion by juangdnar** [filamentphp/filament#9699] [Help] [answered]
is there a way to publish the breadcrumbs.blade.php from the vendor folder, in order to customize it. Thx in advanced

**Comment by leandrocfe** [comment]
breadcrumbs is a component rendered here `vendor/filament/filament/resources/views/components/header/index.blade.php`
Would you like to customize the css?

**Reply by juangdnar** [reply]
I need to place a custom link between the breadcrums and the title of the page, that's why I wanted to know if it is possible to publish that file from the vendor to customize it

**Reply by leandrocfe** [reply, marked as answer]
You could create a custom view in `resources/views/vendor/filament-panels/components/header/index.blade.php`, copy the original content and edit what you want.

**Reply by leandrocfe** [reply]
This can be dangerous: https://filamentphp.com/community/danharrin-publishing-views-in-laravel
